test/shared/addresses.ts

```diff
@@ -0,0 +1,14 @@
+export const addresses = {
```

Comment:
lol are we mainnet forking again?

Reply:
Hah, I was going to but decided against it. I'll probably remove these.